proto/sge/house/query.proto

```diff
@@ -19,19 +19,19 @@ service Query {
 
   // Deposits queries all deposits.
   rpc Deposits(QueryDepositsRequest) returns (QueryDepositsResponse) {
-    option (google.api.http).get = "/sge/deposits";
+    option (google.api.http).get = "/sge/house/deposits";
   }
 
-  // DepositorDeposits queries deposits info for given depositor.
-  rpc DepositorDeposits(QueryDepositorDepositsRequest)
-      returns (QueryDepositorDepositsResponse) {
-    option (google.api.http).get = "/sge/deposits/{depositor_address}";
+  // DepositsByUser queries deposits info for given user.
+  rpc DepositsByUser(QueryDepositsByUserRequest)
```

Comment:
In Cosmos App, "user" is not a industry word, I think the `user` can be replaced with 'account' or `acc` or `depositor`

Reply:
Modified everywhere accordingly @scorpioborn